src/client/extension.ts

```diff
@@ -205,6 +209,8 @@ export async function activate(
                     AadIdObject = pacActiveAuth.Results?.filter(obj => obj.Key === AadIdKey);
                     EnvID = pacActiveAuth.Results?.filter(obj => obj.Key === EnvIdKey);
                     TenantID = pacActiveAuth.Results?.filter(obj => obj.Key === TenantIdKey);
+                    const authInfo = extractAuthInfo(pacActiveAuth.Results);
+                    pacAuthManager.setAuthInfo(authInfo);
```

Comment:
Since we are now setting the auth info here, I guess we can remove it from `ActionsHub.initialize`. What do you think?